### ISSUE TITLE
Add option to remove "clear" functionality from UI

### ISSUE THREAD

**Issue by kiu** [sj26/mailcatcher#489]
We are using mailcatcher in a multi-user environment and sometimes people confuse the "clear" button with "clearing my search" and therefore kill all emails.

It would help if this button could be disabled (in combination with message limit) by a command line option.

**Comment by sj26** [comment]
Are you just wanting to prevent accidental usage?

The clear button does ask for confirmation with a pretty clear message:

> You will lose all your received messages.
> 
> Are you sure you want to clear all messages?

**Comment by rustysponge** [comment]
Yes, but nevertheless it happens from time to time in our setup.

Therefore, in my opinion, a clearer message on the button and/or another button location would be helpful. I think the main problem is that material design (and others) suggests/uses to use a "clear this input" trailing icon on the right side of the input itself and that people are used to this pattern...

**Comment by kiu** [comment]
I have an additional perspective to @rustysponge.

I believe its natural behaviour for a user when he has a task requiring him to use mailcatcher, that he clears all emails to have a fresh start with less clutter. That is cool in a single user environment.

In a multi user environment this user might delete all mails to declutter his experience, but having done this, might have deleted mails other users spent significant time on preparing scenarios for. Rather than solving this through processes (weekly memo: please everyone, do not clear the mailcatcher, think about the needs of your colleagues) a functionality preventing the user from doing so would solve that automatically.

Therefore from my perspective there are two parts. One is unintentional use of the clear functionality and the other is unintentional side effects of using the clear button.

For our org something like this would do the trick:

mailcatcher --messages-limit 500 --no-clear
